web/app/models.py

```diff
@@ -48,6 +49,15 @@ class User(db.Model, UserMixin):
     login_count = db.Column(db.Integer)
     players = relationship("Player")
 
+    # For every AGA member, a fake user (aga_id, claimed=False) is created.
+    # (This fake user holds onto their real life game history for them.)
+    # When that AGA member signs up, the fake user is updated to 
+    # (aga_id, claimed=True), and the real user gets (aga_id, claimed=False).
+    # Thus, only one fake and one real user can ever exist for an aga_id.
```

Comment:
In this case... I'm not sure what the structure of the existing games DB is, but I'm imagining that the real-life games DB has entries under both `old_aga_id` and `vanity_aga_id`. In this case, the real user would have `aga_id=vanity_aga_id`, and the real user would get associated with two fake players, holding games from the old and vanity AGA id. If Davis's script is convenient enough to merge the players as he uploads them, then we don't have to worry about this.